apps/preview/next/pages/showcases/Filters/FiltersPanel.tsx

```diff
@@ -170,10 +171,24 @@ const filtersData: Node[] = [
       { id: 'pr5', label: '$200.00 and above', value: 'above', counter: 18 },
     ],
   },
+  {
+    id: 'acc6',
+    summary: 'Rating',
+    type: 'rating',
+    details: [
+      { id: 'r1', label: '5', value: '5', counter: 10 },
+      { id: 'r2', label: '4 & up', value: '4', counter: 123 },
+      { id: 'r3', label: '3 & up', value: '3', counter: 12 },
+      { id: 'r4', label: '2 & up', value: '2', counter: 3 },
+      { id: 'r5', label: '1 & up', value: '1', counter: 13 },
+    ],
+  },
 ];
 
 export default function FiltersPanel() {
   const [selectedFilters, setSelectedFilters] = useState<string[]>([]);
+  const [checkedPriceState, setCheckedPriceState] = useState<string | null>(null);
+  const [checkedRatingState, setCheckedRatingState] = useState<string | null>(null);
```

Comment:
You can call these states simpler like: `[price, setPrice]` and `[rating, setRating]`.